accounts/abi/bind/base_test.go

```diff
@@ -40,22 +40,22 @@ type mockCaller struct {
 	pendingCallContractCalled bool
 }
 
-func (mc *mockCaller) CodeAt(ctx context.Context, contract common.Address, blockNumber *big.Int) ([]byte, error) {
+func (mc *mockCaller) CodeAt(_ context.Context, _ common.Address, blockNumber *big.Int) ([]byte, error) {
```

Comment:
Please don't use anonymous parameters.

Reply:
I'll revise the change, is there a why not though? It might be helpful for me in future

Reply:
We want to maintain consistent code across the code base. Thats why we decided not to use anonymous parameters even though golang allows for them.